packages/salesforcedx-vscode-core/package.json

```diff
@@ -344,14 +344,7 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "%feature_previews_title%",
-      "properties": {
-        "sfdx.refreshSObjects.enabled": {
-          "type": "boolean",
-          "default": false,
-          "description": "%enable_refresh_description%"
-        }
-      }
+      "title": "%feature_previews_title%"
```

Comment:
If there's no properties, the section doesn't appear in the preferences page. Keeping this so it's easier to re-enable in the future.